README.md

```diff
@@ -26,7 +26,8 @@ Then open the project with VSCode.
 ### Configuration of the application
 
 The example application should first be configured to set at least:
-- `MENDER_SERVER_TENANT_TOKEN` to set the Tenant Token of your account on "https://hosted.mender.io" server;
+- `CONFIG_MENDER_SERVER_HOST` to set the server host. Default is valid for US `https://hosted.mender.io` and should be changed to `https://eu.hosted.mender.io` for EU/UK
```

Comment:
Using the mender US server from Europe is working. The goal of this example is to be very simple and it's not a generic mender tutorial. Mender documentation is there for this purpose.

Reply:
Easy enough - I'll do my own example for the non-American parts of the world 🤣 Laterz

Reply:
Not sure of the goal of your answer, but you can do what you want in your own fork. Just for your information I'm French and living in France ;-)

Reply:
Yeah I just want something that works :) Will do